README.md

```diff
@@ -90,7 +92,7 @@ class InvoicePaid extends Notification
 }
 ```
 
-Here's a screenshot preview of the above notification on Facebook Messenger:
+The notification is being sent to the Facebook page you have created before and you will find the message inside the chat window. Here's a screenshot of how this looks:
```

Comment:
Hmm right, I just want to point out from whom the message ist coming. What about that:

The notification is being sent to the Facebook user you have chosen and the sender will be the Facebook page you have created before. Here's a screenshot preview of the notification inside the chat window.